File: DWR C# Client/DWR/DWR/Model/LoginStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DWR.Model
{
    public class LoginStatus
    {
        public String status { get; set; }
        public String sessionId { get; set; }
        public String username { get; set; }

        public LoginStatus() { 
        
        }

        public LoginStatus(String _status, String _sessionId, String _username) {
            this.status = _status;
            this.sessionId = _sessionId;
            this.username = _username;
        }
    }
}

File: DWR Abgabe Ostern 18.03.2016/DWR_WebServer/src/pkgModel/LoginStatus.java
package pkgModel;

//import java.sql.Date;

import javax.xml.bind.annotation.XmlRootElement;
//import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;

@XmlRootElement (name = "DWR")
public class LoginStatus {
	
	private static final long serialVersionUID = 1L;
	private String status;
	private String username;
	private String sessionId;
	
	/*@XmlJavaTypeAdapter(SqlDateAdapter.class)
	private Date lastLogin;*/
	
	
	public LoginStatus() {
		super();
	}
	
	public LoginStatus(String status, String username, String sessionId) {
		super();
		this.status = status;
		this.username = username;
		this.sessionId = sessionId;
		//this.lastLogin = lastLogin;
	}
	@Override
	public String toString() {
		return "LoginStatus [status=" + status + ", username=" + username
				+ ", sessionId=" + sessionId + "]";/*, lastLogin=" + lastLogin + "]";*/
				
	}
	public String getStatus() {
		return status;
	}
	public void setStatus(String status) {
		this.status = status;
	}
	public String getUsername() {
		return username;
	}
	public void setUsername(String username) {
		this.username = username;
	}
	public String getSessionId() {
		return sessionId;
	}
	public void setSessionId(String sessionId) {
		this.sessionId = sessionId;
	}
	/*public Date getLastLogin() {
		return lastLogin;
	}
	public void setLastLogin(Date lastLogin) {
		this.lastLogin = lastLogin;
	}*/
}
